pages/terms_of_service.mdx

```diff
@@ -0,0 +1,19 @@
+# Terms of Service
+
+These are our terms of service. In case you have any questions or concerns please reach out to us.
```

Comment:
Probably we should add "By purchasing a ticket and attending the conference you agreeing to the following terms"?